spec/lib/jobs/obm-control-spec.js

```diff
@@ -151,6 +182,19 @@ describe("Job.Obm.Node", function () {
                 expect(ObmService.prototype.reboot).to.have.been.calledOnce;
             });
         });
+
+        it('should fail to run OBM command if no target or nodeId was specified', function() {
+
+            // local options with no nodeId field
+            var options = {
+                action: 'reboot',
+                obmServiceName: 'ipmi-obm-service'
+            };
+
+            expect(function() {
+                return new Job(options, {}, uuid.v4());
+            }).to.throw('nodeId (string) is required');
```

Comment:
Thanks.  I'm not sure that looking for any AssertionError is the right approach.  Modifying the unit test is not necessarily a bad thing if the behavior has actually changed.  Expecting AssertionError is too generic and would not actually verify that the specific condition that we are looking for.